.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "wasmCloud",
+    "image": "rust:bullseye",
+    "features": {
+        "devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-binstall:latest": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-watch:latest": {},
```

Comment:
(original: https://github.com/wasmCloud/wasmCloud/pull/335#discussion_r1203944859)

> Why is this necessary?
This allows for running builds or tests continuously, whenever a file changes, using [cargo watch](https://crates.io/crates/cargo-watch)
It is a useful development tool, and is recommended in the wasmCloud/wash repository here: https://github.com/wasmCloud/wash/blob/main/docs/DEVELOPMENT.md

Reply:
Again, it's not required for development, so I don't think it belongs here

Reply:
removed